src/biotite/structure/graphics/rna.py

```diff
@@ -0,0 +1,260 @@
+            'RNAplot is not installed at the specified location, unable to '
+            'plot secondary structure.'
+        )
+
+    # Get the unknotted base pairs
+    if pseudoknot_order is None:
+        # Get a random unknotted structure retaining the maximum number
+        # of base pairs.
+        pseudoknot_order = pseudoknots(base_pairs, max_pseudoknot_order=0)[0]
+    unknotted_base_pairs = base_pairs[pseudoknot_order == 0]
+
+    # If `bond_linewidth` is a single value, extrapolate
+    if isinstance(bond_linewidth, int) or isinstance(bond_linewidth, float):
+        bond_linewidth = np.full(base_pairs.shape[0], bond_linewidth)
+
+    # If `bond_color` is not an array, extrapolate
+    if not isinstance(bond_color, np.ndarray):
+        bond_color = np.full(base_pairs.shape[0], bond_color)
+
+    # Set the default properties of the Matplotlib `bbox` surrounding
+    # the base labels
+    if base_box is None:
+        base_box=np.full(length, {'pad': 0, 'color': 'white'})
+    # if `base_box` is a dictionary, extrapolate
+    elif isinstance(base_box, dict):
+        base_box = np.full(length, base_box)
+
+    # By default pseudoknotted bonds are denoted as dashed lines, while
+    # unknotted bonds are denoted as solid lines
+    if bond_linestyle is None:
+        bond_linestyle = np.full(base_pairs.shape[0], 'solid', dtype='object')
+        bond_linestyle[pseudoknot_order != 0] = 'dashed'
+    # If `bond_linestyle` is a string, extrapolate
+    elif isinstance(bond_linestyle, str):
+        bond_linestyle = np.full(base_pairs.shape[0], bond_linestyle)
+
+    # If pseudoknots are not to be drawn, remove pseudoknotted bonds,
+    # regardless of the given linestyles
+    if not draw_pseudoknots:
+        bond_linestyle[pseudoknot_order != 0] = 'None'
+
+    # Set the default font properties of the base labels
+    if base_font is None:
+        base_font={'size': 'small'}
+
+    # Set the default font properties of the base annotations
+    if annotation_font is None:
+        annotation_font={'size': 'smaller'}
+
+    # If no specific annotation positions are given, annotate every
+    # second base pair
+    if annotation_positions is None:
+        annotation_positions = range(0, length, 2)
+
+    # Get coordinates for secondary structure plot
+    coordinates = RNAplotApp.compute_coordinates(
+        base_pairs=unknotted_base_pairs,
+        length=length,
+        bin_path=bin_path,
+        layout_type=layout_type
+    )
+
+    # Rotate Coordinates
+    if angle != 0:
+        angle = np.deg2rad(angle)
+        rot_matrix = np.array(
+            [[np.cos(angle), -np.sin(angle)],
+            [np.sin(angle),  np.cos(angle)]]
+        )
+        for i, coord in enumerate(coordinates):
+            coordinates[i] = np.dot(rot_matrix, coord)
+
+    # Create Plot
+    fig, ax = plt.subplots(figsize=(10, 10))
+
+    # Remove axes and frame
+    ax.set_xticks([])
+    ax.set_yticks([])
+    plt.box(False)
```

Comment:
I think `plt.box()` won't properly work, in case there are multiple created axes. Alternatively you can use `axes.set_frame_on(False)` and `axes.set_frame_on(False)` and `axes.xaxis/yaxis.set_tick_params(top=False, bottom=False, labeltop=False, labelbottom=False)`